packages/api-plugin-fulfillment/src/xforms/id.js

```diff
@@ -0,0 +1,18 @@
+import decodeOpaqueIdForNamespace from "@reactioncommerce/api-utils/decodeOpaqueIdForNamespace.js";
+import encodeOpaqueId from "@reactioncommerce/api-utils/encodeOpaqueId.js";
+
+const namespaces = {
+  Shop: "reaction/shop",
+  Cart: "reaction/cart",
+  FulfillmentGroup: "reaction/fulfillmentGroup",
+  FulfillmentMethod: "reaction/fulfillmentMethod"
+};
+
+export const encodeShopOpaqueId = encodeOpaqueId(namespaces.Shop);
+export const encodeFulfillmentMethodOpaqueId = encodeOpaqueId(namespaces.FulfillmentMethod);
+export const encodeFulfillmentGroupOpaqueId = encodeOpaqueId(namespaces.FulfillmentGroup);
+
+export const decodeShopOpaqueId = decodeOpaqueIdForNamespace(namespaces.Shop);
```

Comment:
@vanpho93  @zenweasel 
I have removed the opaqueId stuff from the new endpoints introduced by this plugin, but retained them for the old endpoints like `updateFulfillmentOptionsForGroup` & `selectFulfillmentOptionForGroup` thinking it may be required by the current storeFront implementations. The above mentioned xforms/id.js is used by those old endpoints.
Should we be removing it from them also?

Reply:
No, keep them for now but pleases mark the code as deprecated

Reply:
Added the check for opaqueId before calling decode
This update is dependent on #6744